docs/guides/ngrok-for-ingress-to-services-in-a-consul-service-mesh-on-kubernetes.md

```diff
@@ -0,0 +1,312 @@
+---
+title: ngrok for Ingress to Services in a Consul Service Mesh on Kubernetes
+---
+
+# Using ngrok Ingress Controller with Hashicorp's Consul Service Mesh
+
+## Introduction
+
+The [ngrok Ingress Controller for Kubernetes](https://github.com/ngrok/kubernetes-ingress-controller) is our official open-source controller for adding public and secure ingress traffic to your k8s services. It works any Cloud, local, or On-Prem Kubernetes cluster to provide ingress to your services no matter what the network configurations are, as long as it has outbound access to the ngrok service. This allows ngrok to be portable and work seamlessly across any different type of infrastructure.
```

Comment:
> your services no matter what the network configurations are, as long as it has outbound access to the ngrok service. 

your services no matter the network configuration, as long as it has outbound access to the ngrok service.